tsconfig.node.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "module": "es2015",
+        "module": "commonjs",
```

Comment:
See what @esheffield did in place of this. You can remove this change, just make sure your package.json merge is correct.

Reply:
Once his prs merged ill rebase and get rid of these changes i made.

Reply:
OK, my testing PRs have been merged.